blosc-sys/build.rs

```diff
@@ -15,6 +15,18 @@ fn main() {
     } else {
         None
     };
+    #[cfg(unix)]
```

Comment:
Why bother with `#[cfg(unix)]`?  Are you trying to use this crate on Windows?

Reply:
This is for detection through `pkg-config`. The crate should ideally be usable on `windows`.